src/definitions_cxx.hpp

```diff
@@ -340,7 +340,7 @@ constexpr int32_t kParamNodeWidth = 3;
 //
 
 // Performance View constant
-constexpr int32_t kNumParamsForPerformance = 16;
+constexpr int32_t kNumParamsForPerformance = 18;
```

Comment:
increase kNumGlobalParamsForAutomation to 25 above (line 331) as well and then we can add these new params to the select encoder scrolling (globalParamsForAutomation) list in automation_view.cpp

```
    // LPF Cutoff, Resonance
    {params::Kind::UNPATCHED_GLOBAL, params::UNPATCHED_LPF_FREQ},
    {params::Kind::UNPATCHED_GLOBAL, params::UNPATCHED_LPF_RES},
    {params::Kind::UNPATCHED_GLOBAL, params::UNPATCHED_LPF_MORPH},    
    // HPF Cutoff, Resonance
    {params::Kind::UNPATCHED_GLOBAL, params::UNPATCHED_HPF_FREQ},
    {params::Kind::UNPATCHED_GLOBAL, params::UNPATCHED_HPF_RES},
    {params::Kind::UNPATCHED_GLOBAL, params::UNPATCHED_HPF_MORPH},
```